vignettes/dm-class-and-basic-operations.Rmd

```diff
@@ -59,9 +60,36 @@ The relevant functions for creating `dm` objects are:
 2. `as_dm()`
 3. `new_dm()`
 
-We will use tables from the popular  [{nycflights13}](https://github.com/hadley/nycflights13) package:
-```{r message=FALSE}
+There are currently three options available for creating a `dm` object.
+To illustrate these, we will now create the same `dm` using each of these.
```

Comment:
To illustrate these options, we will now create the same `dm` in three different ways.